src/boost.beast/main.cpp

```diff
@@ -0,0 +1,37 @@
+#include <boost/beast/core.hpp>
+#include <boost/beast/http.hpp>
+#include <boost/beast/version.hpp>
+#include <boost/asio/dispatch.hpp>
+#include <boost/asio/strand.hpp>
+#include <cstdlib>
+#include <iostream>
+#include <memory>
+#include <string>
+#include <thread>
+#include <vector>
+#include "listener.h"
+
+
+int main()
+{
+    auto const address = net::ip::make_address("0.0.0.0");
+    auto const port = static_cast<unsigned short>(8080);
```

Comment:
auto const port = std::uint16_t{8080}; ...? no cast needed

Reply:
or simply `uint16_t const port = 8080;`  - you don't use auto for the io_context and vector below, so no need to be picky with AAA

Reply:
done